lib/moped/connection.rb

```diff
@@ -34,9 +34,10 @@ def alive?
     # @since 1.0.0
     def connect
       @sock = if !!options[:ssl]
-        Sockets::SSL.connect(host, port, timeout)
+        ssl_options = options.reject { |k, v| !k.to_s.start_with?('ssl') }
```

Comment:
how about inverting this to be `options.select { |k, v| k.to_s.start_with?('ssl') }` to eliminate the double negative?